---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Gene expression profiles complement the analysis of genomic modifiers of the
  clinical onset of Huntington disease
subtitle: ''
summary: ''
authors:
- Galen E. B. Wright
- Nicholas S. Caron
- Bernard Ng
- Lorenzo Casal
- William Casazza
- Xiaohong Xu
- Jolene Ooi
- Mahmoud A. Pouladi
- Sara Mostafavi
- Colin J. D. Ross
- Michael R. Hayden
tags: []
categories: []
date: '2020-08-01'
lastmod: 2022-12-11T16:52:14-06:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2022-12-11T22:52:14.416187Z'
publication_types:
- '2'
abstract: Huntington disease (HD) is a neurodegenerative disorder that is caused by
  a CAG repeat expansion in HTT. The length of this repeat, however, only explains
  a proportion of the variability in age of onset in patients. Genome-wide association
  studies have identified modifiers that contribute toward a proportion of the observed
  variance. By incorporating tissue-specific transcriptomic information with these
  results, additional modifiers can be identified. We performed a transcriptome-wide
  association study assessing heritable differences in genetically determined expression
  in diverse tissues, with genome-wide data from over 4000 patients. Functional validation
  of prioritized genes was undertaken in isogenic HD stem cells and patient brains.
  Enrichment analyses were performed with biologically relevant gene sets to identify
  the core pathways. HD-associated gene coexpression modules were assessed for associations
  with neurological phenotypes in an independent cohort and to guide drug repurposing
  analyses. Transcriptomic analyses identified genes that were associated with age
  of HD onset and displayed colocalization with gene expression signals in brain tissue
  (FAN1, GPR161, PMS2, SUMF2), with supporting evidence from functional experiments.
  This included genes involved in DNA repair, as well as novel-candidate modifier
  genes that have been associated with other neurological conditions. Further, cortical
  coexpression modules were also associated with cognitive decline and HD-related
  traits in a longitudinal cohort. In summary, the combination of population-scale
  gene expression information with HD patient genomic data identified novel modifier
  genes for the disorder. Further, these analyses expanded the pathways potentially
  involved in modifying HD onset and prioritized candidate therapeutics for future
  study. [GRAPHICS] . Schematic representation of methodology employed to identify
  transcriptomic modifiers of Huntington disease (HD). Transcriptome-wide association
  analyses facilitated (A) the identification of top transcriptomic HD onset modifier
  genes, (B) relevant biologically diverse gene sets that are enriched for this trait
  and (C) human cortical coexpression modules that are relevant for HD onset and related
  phenotypes in an aging cohort, along with signature matching of HD signals for drug
  repurposing.
publication: '*HUMAN MOLECULAR GENETICS*'
doi: 10.1093/hmg/ddaa184
---
