---
# Documentation: https://wowchemy.com/docs/managing-content/

title: The Extent and Impact of Variation in ADME Genes in Sub-Saharan African Populations
subtitle: ''
summary: ''
authors:
- Jorge E. B. da Rocha
- Houcemeddine Othman
- Gerrit Botha
- Laura Cottino
- David Twesigomwe
- Samah Ahmed
- I, Britt Drogemoller
- Faisal M. Fadlelmola
- Philip Machanick
- Mamana Mbiyavanga
- Sumir Panji
- Galen E. B. Wright
- Clement Adebamowo
- Mogomotsi Matshaba
- Michele Ramsay
- Gustave Simo
- Martin C. Simuunza
- Caroline T. Tiemessen
- Sandra Baldwin
- Mathias Chiano
- Charles Cox
- Annette S. Gross
- Pamela Thomas
- Francisco-Javier Gamo
- Scott Hazelhurst
tags:
- ADME; genetic diversity; Africa; pharmacogenomics; CNV
categories: []
date: '2021-04-01'
lastmod: 2022-12-11T16:52:12-06:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2022-12-11T22:52:12.381571Z'
publication_types:
- '2'
abstract: 'Introduction: Investigating variation in genes involved in the absorption,
  distribution, metabolism, and excretion (ADME) of drugs are key to characterizing
  pharmacogenomic (PGx) relationships. ADME gene variation is relatively well characterized
  in European and Asian populations, but data from African populations are under-studied-which
  has implications for drug safety and effective use in Africa.Results: We identified
  significant ADME gene variation in African populations using data from 458 high-coverage
  whole genome sequences, 412 of which are novel, and from previously available African
  sequences from the 1,000 Genomes Project. ADME variation was not uniform across
  African populations, particularly within high impact coding variation. Copy number
  variation was detected in 116 ADME genes, with equal ratios of duplications/deletions.
  We identified 930 potential high impact coding variants, of which most are discrete
  to a single African population cluster. Large frequency differences (i.e., >10%)
  were seen in common high impact variants between clusters. Several novel variants
  are predicted to have a significant impact on protein structure, but additional
  functional work is needed to confirm the outcome of these for PGx use. Most variants
  of known clinical outcome are rare in Africa compared to European populations, potentially
  reflecting a clinical PGx research bias to European populations.Discussion: The
  genetic diversity of ADME genes across sub-Saharan African populations is large.
  The Southern African population cluster is most distinct from that of far West Africa.
  PGx strategies based on European variants will be of limited use in African populations.
  Although established variants are important, PGx must take into account the full
  range of African variation. This work urges further characterization of variants
  in African populations including in vitro and in silico studies, and to consider
  the unique African ADME landscape when developing precision medicine guidelines
  and tools for African populations.'
publication: '*FRONTIERS IN PHARMACOLOGY*'
doi: 10.3389/fphar.2021.634016
---
