---
# Documentation: https://wowchemy.com/docs/managing-content/

title: Integrating disease and drug-related phenotypes for improved identification
  of pharmacogenomic variants
subtitle: ''
summary: ''
authors:
- Tom W. Ouellette
- Galen E. B. Wright
- I, Britt Droegemoeller
- Colin J. D. Ross
- Bruce C. Carleton
tags:
- adverse drug reactions; data integration; disease-drug relationships; drug response;
  pleiotropy
categories: []
date: '2021-04-01'
lastmod: 2022-12-11T16:52:13-06:00
featured: false
draft: false

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder.
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["internal-project"]` references `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects: []
publishDate: '2022-12-11T22:52:12.581122Z'
publication_types:
- '2'
abstract: "Lay abstract Disease-focused genome-wide association studies (GWAS) have\
  \ identified a plethora of actionable genetic variants over the last 20 years. International\
  \ collaboration and technological breakthroughs have enabled rapid genotyping and\
  \ data collection, which has correspondingly increased sample size and power. Contrastingly,\
  \ recruitment of well-characterized cohorts of patients for pharmacogenomics research\
  \ has proven challenging. Given the greater number of associated genetic variants\
  \ and larger cohort sizes in common disease GWAS, we hypothesized that integrating\
  \ genome-wide disease and pharmacogenomic data may drive new insights into drug\
  \ toxicity and drug efficacy phenotypes, beyond the standard scope of current pharmacogenetic\
  \ analyses. Using GWAS summary statistics from the UK Biobank, European Bioinformatics\
  \ Institute-GWAS catalog, and the Pharmacogenomics Research Network, and a methodological\
  \ framework incorporating colocalization analysis, we validated pleiotropy at the\
  \ ABCG2 locus between allopurinol response, gout, and serum urate and identified\
  \ novel pleiotropy between antihypertensive-induced new-onset diabetes, Crohn's\
  \ disease and inflammatory bowel disease at the IL18RAP/SLC9A4 locus. Aim: To improve\
  \ the identification and interpretation of pharmacogenetic variants through the\
  \ integration of disease and drug-related traits. Materials & methods: We hypothesized\
  \ that integrating genome-wide disease and pharmacogenomic data may drive new insights\
  \ into drug toxicity and response by identifying shared genetic architecture. Pleiotropic\
  \ variants were identified using a methodological framework incorporating colocalization\
  \ analysis. Results: Using genome-wide association studies summary statistics from\
  \ the UK Biobank, European Bioinformatics Institute-genome-wide association studies\
  \ catalog and the Pharmacogenomics Research Network, we validated pleiotropy at\
  \ the ABCG2 locus between allopurinol response and gout and identified novel pleiotropy\
  \ between antihypertensive-induced new-onset diabetes, Crohn's disease and inflammatory\
  \ bowel disease at the IL18RAP/SLC9A4 locus. Conclusion: New mechanistic insights\
  \ and genetic loci can be uncovered by identifying pleiotropy between disease and\
  \ drug-related traits."
publication: '*PHARMACOGENOMICS*'
doi: 10.2217/pgs-2020-0130
---
